_data/entertainment.yml

```diff
@@ -27,6 +27,14 @@ websites:
       img: deezer.png
       tfa: No
 
+    - name: Filmweb
+      url: https://www.filmweb.pl
+      twitter: filmweb
+      facebook: filmweb
+      img: filmweb.pl
```

Comment:
There's a typo here ;-)

Reply:
```suggestion
      img: filmweb.png
```

Reply:
@einsteinsfool As you didn't allow changes to your PR from other members, you'd need to change this yourself :)
// Kai

Reply:
My bad, sorry. I guess I'm too used to writing the URL. Fixed. :-)